include/IECoreHoudini/SOP_SceneCacheSource.h

```diff
@@ -97,6 +98,7 @@ class IECOREHOUDINI_API SOP_SceneCacheSource : public SceneCacheNode<SOP_Node>
 		// Convert the object to Houdini, optimizing for animated primitive variables if possible.
 		bool convertObject( const IECore::Object *object, const std::string &name, const IECoreScene::SceneInterface *scene, Parameters &params );
 
+		void invisibleLocations( const IECoreScene::SceneInterface *scene, bool &invisibleLocations, double time );
```

Comment:
Why not return the result instead of having a void return and an output parameter?

Reply:
good point, I fixed this in place ( same commit ).